src/config/scrivitoContentBrowser.js

```diff
@@ -1,107 +1,152 @@
+  };
+}
+
+function defaultFilter() {
+  return {
+    _objClass: {
+      options: {
+        All: {
+          title: "All",
+          icon: "folder",
+          query: Scrivito.Obj.all(),
+          selected: true,
         },
+        ...{ Image: descriptionForObjClass("Image") },
         Pages: {
           title: "Pages",
           icon: "sheet",
           field: "_objClass",
-          value: [
-            "Author",
-            "Blog",
-            "BlogPost",
-            "Event",
-            "Homepage",
-            "Job",
-            "LandingPage",
-            "Page",
-            "Redirect",
-            "SearchResults",
-          ],
-          options: {
-            Page: {
-              title: "Standard pages",
-              icon: "sheet",
-            },
-            LandingPage: {
-              title: "Landing pages",
-              icon: "inbox",
-            },
-            BlogPost: {
-              title: "Blog posts",
-              icon: "pen",
-            },
-            Author: {
-              title: "Authors",
-              icon: "user",
-            },
-            Event: {
-              title: "Events",
-              icon: "cal",
-            },
-            Job: {
-              title: "Jobs",
-              icon: "suitcase",
-            },
-            Redirect: {
-              title: "Redirects",
-              icon: "link",
-            },
-            Homepage: {
-              title: "Homepage",
-              icon: "inbox",
-            },
-            Blog: {
-              title: "Blog",
-              icon: "pen",
-            },
-            SearchResults: {
-              title: "Search results",
-              icon: "lens",
-            },
-          },
-        },
-        Download: {
-          title: "Downloads",
-          icon: "pdf",
-          field: "_objClass",
-          value: "Download",
-        },
-        Video: {
-          title: "Videos",
-          icon: "video",
-          field: "_objClass",
-          value: "Video",
+          value: PAGES,
+          options: PAGES.reduce((interimOptions, item) => {
+            return {
+              ...interimOptions,
+              ...{ [item]: descriptionForObjClass(item) },
+            };
+          }, {}),
         },
+        ...{ Download: descriptionForObjClass("Download") },
+        ...{ Video: descriptionForObjClass("Video") },
       },
     },
-    _modification: {
-      title: "Changed",
-      type: "checkbox",
-      expanded: true,
-      field: "_modification",
-      options: {
-        New: {
-          value: "new",
-        },
-        Edited: {
-          value: "edited",
-        },
-      },
+    _modification: MODIFICATION_FILTER,
+  };
+}
+
+function descriptionForObjClass(objClass) {
+  const description = DESCRIPTIONS[objClass] || {
+    title: objClass,
+    icon: "question",
+  };
+  return { field: "_objClass", value: objClass, ...description };
+}
+
+const DESCRIPTIONS = {
```

Comment:
* [ ] The term "description" is misleading. In the docs only the `title` is a description. Everything else is just called "options". To prevent confusion, we should name it "options", or if it must be more specific something picked from this set: "obj class filter presentation options".

Reply:
May I add "label" to the set? :-)

Reply:
Whatever 😛. As long we don't contradict https://www.scrivito.com/js-sdk/configureContentBrowser#several%20options

Reply:
How about `filterOptionForObjClass` and `FILTER_OPTIONS`?

Reply:
`filterOptionsForObjClass` (+`s`) and `FILTER_OPTIONS` LGTM